src/steps/ms-defender/finding/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CveRelationshipsFindings Should create a Vulnerability finding entity correctly when there is a CVE relationship: cveFindingEntitiesSuccessful 1`] = `Array []`;
+
+exports[`fetchFindings Should create a Vulnerability finding entity correctly when user has the correct permissions: vulnerabilityFindingEntitiesSuccessful 1`] = `Array []`;
```

Comment:
Another case where I'm surprised to find there is no data in these snapshots. Is it really the expectation that there would be nothing?